app/controllers/change_controller.rb

```diff
@@ -74,7 +74,7 @@ def update
   end
 
   def post_to_register(register_name, rsf_body)
-    protocol = Rails.configuration.register_ssl ? protocol = 'https' : protocol = 'http'
+    protocol = Rails.configuration.register_ssl ? 'https' : 'http'
```

Comment:
This has been tested?

Reply:
I have tested the syntax in IRB. This was a suggested refactor in RubyMine as protocol is needlessly redeclared.

Reply:
Makes total sense, just checking with my limited experience of Ruby.

Reply:
This is legit. +1

Reply:
This works fine